auth/builtin.go

```diff
@@ -199,3 +188,14 @@ func (r *Builtin) NewToken(user string, scopes []string, claims jwt.MapClaims) (
 	signed, err = token.SignedString([]byte(Settings.Auth.Token.Key))
 	return
 }
+
+// parseToken returns the token
```

Comment:
**hint**: No reason for parseToken() to be a package function. Consistent with the project, this should be a method on Builtin.
Simpler for parameter to be a request.